### DISCUSSION TITLE
ability or button to have calibration completly restart with empty values

### DISCUSSION THREAD

**Discussion by ecto1a2003** [ESPresense/ESPresense-companion#585] [Ideas]
My setup is weird rn and has sensor moving all over and i can fix it uless i blank out the calibration and the values start to normalize again

**Comment by supaeasy** [comment]
Simply reset absorption to standard values (3.5) or temporarily set the calibration interval to some seconds.

**Comment by ecto1a2003** [comment]
Yes, but a way to do it enmase is what i was hoping for

**Reply by supaeasy** [reply]
Changing the interval IS en masse. Instant and for all.

**Comment by ecto1a2003** [comment]
But id like to start from 0 and just do it with a button

**Comment by RaoulSargent** [comment]
Perhaps an example or explanation as to “how” to reset the absorption or change the interval would provide the required reassurance that it is easy to do.

**Comment by ecto1a2003** [comment]
I know how, but instead of going to each node and blanking by hand a simple button that does if for me would be great

**Comment by 51av0sh** [comment]
Looks like this was implemented but the "Reset Calibration" button on the Calibration screen just gives a 404
![Screenshot 2024-10-21 at 9 47 09 AM](https://github.com/user-attachments/assets/3a8ddb37-9f01-4e9f-9cf1-7ee39111e1a0)

**Reply by DTTerastar** [reply]
Sorry, It worked just fine without the HASS proxy and I didn't realize it was broke until I read this! Fixed in 1.0.4
